docs/ref/templates/builtins.txt

```diff
@@ -699,6 +699,14 @@ available to the included template::
 
     {% include "name_snippet.html" with greeting="Hi" only %}
 
+If the included template is missing or has syntax errors, the behavior varies
+depending on the template engine's ``debug`` option (if not set, this option
+defaults to the value of :setting:`DEBUG`).
+
+When debug mode is turned on, :exc:`~django.template.TemplateDoesNotExist` or
+:exc:`~django.template.TemplateSyntaxError` exception will be raised;
+otherwise {% include %} silences the exception and returns an empty string.
```

Comment:
Please combine paragraphs and add double backticks around {% include %} for code block highlighting.